### PR TITLE
Fix/108300 date time block fixes

### PR DESCRIPTION
**Pull request by paulmskim**
For ticket: https://central.tri.be/issues/108300

### CODE REVIEW

**Comment by mitogh** [comment]
On hold due: 

- https://github.com/moderntribe/events-gutenberg/pull/124#issuecomment-401234358

**Comment by paulmskim** [comment]
@mitogh could you review this when you get a chance? thanks!

**Comment by mitogh** [comment]
Seems like you have a conflict to solve on `readme.md` @paulmskim

File: src/modules/blocks/event-datetime/block.js [hunk 1]
```diff
@@ -202,6 +219,11 @@ class EventDateTime extends Component {
 			.isSame( toMoment( end || this.props.end ), 'day' );
 	}
 
+	isSameYear( start, end ) {
```
Comment (mitogh) [inline src/modules/blocks/event-datetime/block.js]: Seems like you can simplify the conditional fallback on the function by using default params

```js
isSameYear( start = this.props.start, end = this.props.end ) {
 ...
}
```

File: src/modules/editor/utils/date.js [hunk 2]
```diff
@@ -12,13 +12,16 @@ export const FORMATS = {
 	TIME: 'HH:mm:ss',
 	DATE_TIME: 'YYYY-MM-DD HH:mm:ss',
 	WP: {
+		...formats,
 		time: 'g:i a',
-		date: 'F j',
+		date: 'F j, Y',
 		datetime: 'F j, Y g:i a',
-		...formats,
+		dateNoYear: 'F j',
```
Comment (mitogh) [inline src/modules/editor/utils/date.js]: We need to overwrite the default with `formats` which are the values defined on the WP editor.

File: src/modules/blocks/event-datetime/block.js (hunk 1 above)
Comment (paulmskim) [reply src/modules/blocks/event-datetime/block.js]: this was basically copied from your `isSameDay()` function, but checking year. I will make this change for both.

**Comment by paulmskim** [comment]
back to you @mitogh

**Comment by mitogh** [comment]
👍 @paulmskim

**Pull request merged**